nix/nft-marketplace-frontend.nix

```diff
@@ -0,0 +1,144 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.nft-marketplace-frontend;
+in
+{
+  options.services.nft-marketplace-frontend = {
+    enable = lib.mkEnableOption "Enable nft-marketplace frontend";
+
+    artifactsBuilder = lib.mkOption {
+      type = with lib.types; functionTo package;
+    };
+
+    artifacts = lib.mkOption {
+      type = lib.types.package;
+      default = with cfg; (artifactsBuilder
+        {
+          SKIP_PREFLIGHT_CHECK = true; # TODO do we really need this?
```

Comment:
The `SKIP_PREFLIGHT_CHECK` is an option for disabling some dependency graph checks I believe (nothing to do with CORS as it may seem). It's something that has little documentation but is recommended for resolving certain build errors.

If you can remove this and everything still builds successfully, I'd say we no longer need it.

Reply:
It seems to build even without (I've added it at a later time). I'm removing it.